.openshift/templates/ninja-board-back-end.yml

```diff
@@ -0,0 +1,220 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: ninja-board-back-end
+  annotations:
+    iconClass: icon-jboss
+    tags: quarkus, java
+    description: Build the back-end Ninja Board App
+labels:
+  application: ${APPLICATION_NAME}
+objects:
+- apiVersion: image.openshift.io/v1
+  kind: ImageStream
+  metadata:
+    labels:
+      app: ${APPLICATION_NAME}
+    name: openjdk-11-rhel7
+  spec:
+    dockerImageRepository: "registry.access.redhat.com/openjdk/openjdk-11-rhel7"
```

Comment:
Can we move this to a parameter?

Reply:
@sabre1041 Yes, I'll move it shortly

Reply:
@sabre1041 I have added it as a parameter.